API.md

```diff
@@ -410,6 +410,7 @@ var expires = new Date
 expires.setSeconds(24 * 60 * 60 * 10) //10 days
 policy.setExpires(expires)
 policy.setContentType("image/png")
+policy.setMaxLength(1024*1024) // Max upload size is 1MB
 ```
```

Comment:
We should implement setContentLength(max, min) just like Golang it is up to the caller to choose.